internal/wstrust/wstrust_mex_document.go

```diff
@@ -129,23 +120,17 @@ func CreateWsTrustMexDocumentFromDef(definitions Definitions) (MexDocument, erro
 				return MexDocument{}, fmt.Errorf("cannot create MexDocument: %w", err)
 			}
 
-			log.Tracef("Associated port '%v' with binding, url '%v'", bindingName, url)
 			switch binding.EndpointType {
 			case wsEndpointTypeUsernamePassword:
-				if updateEndpoint(&usernamePasswordEndpoint, endpoint) {
```

Comment:
looks like updateEndpoint can be simplified a bit if it doesn't need to return a bool